dev/var_names_xportr.Rmd

```diff
@@ -80,10 +80,11 @@ In order to make use of the functions within `xportr` you will need to create an
 
 ```{r}
 var_spec <- readxl::read_xlsx(
-  system.file(paste0("specs/", "ADaM_admiral_spec.xlsx"), package = "xportr"), sheet = "Variables") %>%
+  system.file(paste0("specs/", "ADaM_admiral_spec.xlsx"), package = "xportr"),
+  sheet = "Variables"
+) %>%
   dplyr::rename(type = "Data Type") %>%
-  rlang::set_names(tolower) 
-  
+  rlang::set_names(tolower)
```

Comment:
could we use dplyr and base?  I always felt rlang was a bit weird here.
 
`dplyr::rename_with(tolower)`

Reply:
Done!